### PR TITLE
Update windows-10-subscription-activation.md

### PR DESCRIPTION
**Pull request by msbemba**
added a note about  excluding the Universal Store Service APIs and Web Application in the device compliance policy reference - https://learn.microsoft.com/en-us/azure/active-directory/conditional-access/howto-conditional-access-policy-compliant-device

Per issue#https://github.com/MicrosoftDocs/windows-itpro-docs/issues/10972

### CODE REVIEW

File: windows/deployment/windows-10-subscription-activation.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ This article covers the following information:
 
 For more information on how to deploy Enterprise licenses, see [Deploy Windows Enterprise licenses](deploy-enterprise-licenses.md).
 
+> [!NOTE]
+> Organizations that use the [Subscription Activation](/windows/deployment/windows-10-subscription-activation) feature to enable users to “step-up” from one version of Windows to another, may want to exclude the Universal Store Service APIs and Web Application, AppID 45a330b1-b1ec-4cc1-9161-9f03992aa49f from their device compliance policy.
```
Comment (JohanFreelancer9) [inline windows/deployment/windows-10-subscription-activation.md:44]:
```suggestion
> Organizations that use the [Subscription Activation](/windows/deployment/windows-10-subscription-activation) feature to enable users to upgrade from one version of Windows to another might want to exclude the Universal Store Service APIs and Web Application, AppID 45a330b1-b1ec-4cc1-9161-9f03992aa49f, from their device compliance policy.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@frankroj Please merge this PR. Thanks

**Comment by frankroj** [comment]
This note previously existed in the docs and then was removed. Verifying with PMs if we should add back in.

**Comment by lee-halford** [comment]
Noting that this has been mentioned in the Conditional Access Documentation which has a reference to this page. But there is no reference on this page about the problem. Would be good to also link back to the CA policy doc as well.

https://github.com/MicrosoftDocs/azure-docs/blob/main/articles/active-directory/conditional-access/howto-conditional-access-policy-all-users-mfa.md#subscription-activation

**Review by ngangulyms — APPROVED** [review]

**Pull request merged**